windows/runner/Runner.rc

```diff
@@ -90,7 +90,7 @@ BEGIN
         BLOCK "040904e4"
         BEGIN
             VALUE "CompanyName", "network.zenon.syrius" "\0"
-            VALUE "FileDescription", "s y r i u s (v0.0.7-alphanet)" "\0"
+            VALUE "FileDescription", "s y r i u s (" VERSION_AS_STRING ")" "\0"
```

Comment:
Does this result in space (" ") padding on each end of the string?

Reply:
Nope.
![image](https://github.com/hypercore-one/syrius/assets/90819192/1ed21da3-c558-4e35-8a38-a9ede92460b3)